### ISSUE TITLE
feat: make admin images thumbnails instead of full size

### ISSUE THREAD

**Issue by andreedd** [Datateknologerna-vid-Abo-Akademi/date-website#179]
**Is your feature request related to a problem? Please describe.**

The small preview pictures in admin collection is atm rendered as full resolution.

**Describe the solution you'd like**

make them render as thumbnails with sorl-thumbnail which we are using in gallery

**Additional context**

**Comment by jaspnas** [comment]
Sorl-thumbnail has been deemed too slow, and image size doesn't seem to be a problem atm. 

Maybe integrate this after #314 has been completed in a new issue
